Project/Sources/Classes/_JWT.4dm

```diff
@@ -9,11 +9,17 @@ property privateKey : Text
 
 Class constructor($inParam : Object)
 	
-	var $alg; $typ : Text
+	var $alg; $typ; $x5t : Text
 	
 	$alg:=(OB Is defined($inParam; "header") && OB Is defined($inParam.header; "alg")) ? $inParam.header.alg : "RS256"
 	$typ:=(OB Is defined($inParam; "header") && OB Is defined($inParam.header; "typ")) ? $inParam.header.typ : "JWT"
+	$x5t:=(OB Is defined($inParam; "header") && OB Is defined($inParam.header; "x5t")) ? $inParam.header.x5t : Null
```

Comment:
if compiled passing `Null` to `Text` var is undefined (could crash I think)

so setting empty string instead`""` instead of Null

and after 
```4d
If (Length($x5t)>0)
```

or do use this local variable Text anymore and do
```4d
If ((OB Is defined($inParam; "header") && OB Is defined($inParam.header; "x5t")))
     This.header.x5t:=$inParam.header.x5t
End if 
```

ps: not for this PullRequest, but just as note, maybe there is better way to check headers without `OB Is defined`